app/Services/TweetService.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace App\Services;
+
+use Abraham\TwitterOAuth\TwitterOAuth;
+
+class TweetService
+{
+    private $client;
+
+    public function __construct()
+    {
+        $consumer_key = env('TWITTER_CONSUMER_KEY');
+        $consumer_secret = env('TWITTER_CONSUMER_SECRET');
+        $access_token = env('TWITTER_ACCESS_TOKEN');
+        $access_secret = env('TWITTER_ACCESS_SECRET');
+
```

Comment:
TwitterOAuth need string type. Why load from env?
```suggestion
        $consumer_key = env('TWITTER_CONSUMER_KEY', '');
        $consumer_secret = env('TWITTER_CONSUMER_SECRET', '');
        $access_token = env('TWITTER_ACCESS_TOKEN', '');
        $access_secret = env('TWITTER_ACCESS_SECRET', '');
```